azurerm/resource_arm_app_service.go

```diff
@@ -496,6 +526,11 @@ func resourceArmAppServiceRead(d *schema.ResourceData, meta interface{}) error {
 		return err
 	}
 
+	authSettings := azure.FlattenAppServiceAuthSettings(authResp.SiteAuthSettingsProperties)
+	if err := d.Set("auth_settings", authSettings); err != nil {
+		return err
```

Comment:
could we wrap this error:

```suggestion
		return fmt.Errorf("Error setting `auth_settings`: %+s", err)
```